### PR TITLE
chore(e2e): add e2e tests for app and svc delete

### PR DESCRIPTION
**Pull request by seongm-1**
Added e2e tests.

Had to reinitialize app and svc inside the test files since my tests revolves around not deleting local files (expect .workspace) when running `app delete` and `svc delete`. 

Addresses #1034 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: e2e/addons/addons_test.go [hunk 1]
```diff
@@ -47,6 +47,17 @@ var _ = Describe("addons flow", func() {
 			Expect(appShowOutput.Name).To(Equal(appName))
 			Expect(appShowOutput.URI).To(BeEmpty())
 		})
+
+		It("app delete does delete .workspace", func() {
+			_, err := cli.AppDelete(map[string]string{"test": "default"})
```
Comment (iamhopaul123) [inline e2e/addons/addons_test.go:52]: lol test environment doesn't exist yet

File: e2e/addons/addons_test.go [hunk 2]
```diff
@@ -47,6 +47,17 @@ var _ = Describe("addons flow", func() {
...
+			Expect(err).NotTo(HaveOccurred())
+			Expect("./copilot/.workspace").ShouldNot(BeAnExistingFile())
+
```
Comment (iamhopaul123) [inline e2e/addons/addons_test.go:55]: maybe validate if `copilot` folder exists as well?

File: e2e/addons/addons_test.go [hunk 3]
```diff
@@ -213,5 +225,21 @@ var _ = Describe("addons flow", func() {
 				Expect(logLine.IngestionTime).NotTo(Equal(0))
 			}
 		})
+
+		It("svc delete should not delete local files", func() {
+			_, err := cli.SvcDelete(svcName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect("./copilot/hello/addons").Should(BeAnExistingFile())
```
Comment (seongm-1) [inline e2e/addons/addons_test.go:232]: The tests passes by using `BeAnExistingFile`, will switch this to `BeADirectory`

File: e2e/addons/addons_test.go (hunk 1 above)
Comment (seongm-1) [reply e2e/addons/addons_test.go]: Good catch!

File: e2e/addons/addons_test.go [hunk 4]
```diff
@@ -47,6 +48,18 @@ var _ = Describe("addons flow", func() {
...
+			_, err := cli.AppDelete(map[string]string{"": ""})
+			Expect(err).NotTo(HaveOccurred())
+			Expect("./copilot").Should(BeADirectory())
+			Expect("./copilot/.workspace").ShouldNot(BeAnExistingFile())
+
+			_, initErr = cli.AppInit(&client.AppInitRequest{
+				AppName: appName,
+			})
+			Expect(initErr).NotTo(HaveOccurred())
+		})
```
Comment (seongm-1) [inline e2e/addons/addons_test.go:62]: I actually want to move this block of code for last and check to see if the manifest.yml and addons/ are still present. Will send another PR out

File: e2e/addons/addons_test.go [hunk 5]
```diff
@@ -33,6 +33,7 @@ var _ = Describe("addons flow", func() {
 
 		It("app init creates an copilot directory", func() {
```
Comment (iamhopaul123) [inline e2e/addons/addons_test.go:34]: maybe "app init creates an copilot directory and a workspace file"? since you validate for the file as well.

File: e2e/addons/addons_test.go [hunk 6]
```diff
@@ -213,5 +216,35 @@ var _ = Describe("addons flow", func() {
...
 		})
+
+		It("svc delete should not delete local files", func() {
+			_, err := cli.SvcDelete(svcName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect("./copilot/hello/addons").Should(BeADirectory())
+			Expect("./copilot/hello/manifest.yml").Should(BeAnExistingFile())
+			Expect("./copilot/.workspace").Should(BeAnExistingFile())
+
+			_, svcInitErr = cli.SvcInit(&client.SvcInitRequest{
```
Comment (efekarakus) [inline e2e/addons/addons_test.go:227]: Can you add a comment on why we're re-creating these resources, eventually we might forget why
